modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/PublisherTelemetryConfiguration.cs

```diff
@@ -97,7 +98,8 @@ public string PatternMatch(string stringToParse)
             Match match = _patternRegex.Match(stringToParse);
             if (match.Groups[0].Success)
             {
-                foreach (var group in match.Groups.Values.Skip(1))
+                IEnumerable<Group> groups = match.Groups;
```

Comment:
For debugging?

Reply:
trick to get access to .skip otherwise build error